tests/integration/list/list_integ_base.py

```diff
@@ -0,0 +1,31 @@
+import os
+from unittest import TestCase
+from pathlib import Path
+import uuid
+import shutil
+import tempfile
+from tests.testing_utils import get_sam_command
+
+
+class ListIntegBase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.cmd = cls.base_command()
+        cls.test_data_path = Path(__file__).resolve().parents[1].joinpath("testdata", "list")
+
+    def setUp(self):
+        super().setUp()
+        self.scratch_dir = str(Path(__file__).resolve().parent.joinpath(str(uuid.uuid4()).replace("-", "")[:10]))
+        shutil.rmtree(self.scratch_dir, ignore_errors=True)
+        os.mkdir(self.scratch_dir)
+
+        self.working_dir = tempfile.mkdtemp(dir=self.scratch_dir)
```

Comment:
What is the reason for creating a folder and removing some?

Reply:
I was referencing build_integ_base.py, and it seems to be done there

Reply:
I see, build tests will require some local file setup, that is why we need such actions there. For this initiative, I think we won't need such requirement (at least for now). For that reason I would remove any unnecessary calls from the tests.

Reply:
Sounds good, removed, thanks